pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -15,10 +15,14 @@ import (
 const (
 	virtualBoxDownloadURL   = "https://download.virtualbox.org/virtualbox/6.0.4/VirtualBox-6.0.4-128413-OSX.dmg"
 	virtualBoxMountLocation = "/Volumes/VirtualBox"
+	DefaultVMDriver         = "virtualbox"
```

Comment:
same here... we can't have this as the current default... as defaults never change over time.